crates/pipeline_manager/src/compiler.rs

```diff
@@ -793,7 +815,8 @@ impl Compiler {
 
 #[derive(Eq, PartialEq)]
 enum Stage {
-    Sql,
+    SqlToRust,
```

Comment:
Does this need UI adjustment

Reply:
This isn't exposed through the API. The set of visible program statuses is still the same, it's just that the state machine goes from CompilingSql to Success directly.